conans/client/rest/remote_credentials.py

```diff
@@ -66,3 +91,18 @@ def _get_env(remote, user):
         if passwd:
             ConanOutput().info("Got password '******' from environment")
         return user, passwd
+
+    def _load_auth_plugin(self):
```

Comment:
Let's have this as a free function, and pass the auth_plugin_path as an argument, instead of saving it to the class